experiments/memcached-sensitivity-profile/docs/installation.md

```diff
@@ -17,85 +17,176 @@
+SUT Node requires following application installed.
 
-**NOTE** Running Cassandra in docker containers is not advised for production environments.
-Additionally, be careful if not used with docker volume mounts as you may experience data loss.
+* Experiment Binaries - *Swan Experiments*
+* Memcached & Best Effort Workloads - *Worklods that will be tested in colocation*
+* Snap - *Intel Telemetry Framework*
+* Hyperkube - *Kubernetes in single binary*
+* Docker - *Container Runtime*
+* Snap Plugins - *Plugins for gathering telemetry & experiment results*
 
-In order to reduce any unintended interference, it is not recommended that any of the machines involved in the experiment (executing memcached or load generators) should also be host for Cassandra. 
+**Load Generator Agent:**
 
-### Snap Service
-Please Install Snap on System Under Test (SUT) machine.
+* Mutilate - *Memcached Load Generator*
 
-Swan uses Snap to collect and store metrics in Cassandra Database. See the [Snap installation guide](https://github.com/intelsdi-x/snap#installation) for guidance of how to configure and install Snap. 
+**Services Node:**
 
-## Binaries Installation
-All of binaries listed here should be available in directory listed in `$PATH` for user `root`. They should be deployed on SUT, except Mutilate which should be deployed on all Load Generator and Services machines.
+* Mutilate - *Master for agent synchronisation*
+* Cassandra - *Database for storing experiment results* 
+* Hyperkube - *Kubernetes Control Plane*
+* Etcd - *Backend for Kubernetes*
 
-### Kubernetes (optional)
-When Experiment is not planned to be run on Kubernetes, or when user provides it's own cluster this step can be skipped. Instruction containing connection to user provided Kubernetes cluster are described in [Kubernetes Flags](swan_flags.md#Kubernetes-Flags) section.
+### The Installation details are as follow:
 
-The Experiment requires Kubernetes in version 1.5.x. 
-Please download binaries from https://github.com/kubernetes/kubernetes/releases, copy hyperkube to directory listed in`$PATH` and run `./hyperkube --make-symlinks` to make it usable by Swan. Hyperkube binary should have executable bit set. Swan expects that Docker will be the default container runtime for Kubernetes. Please make sure that all dependencies for Kubernetes (e.g. Docker, Etcd) are running and in proper versions.
+`wget` must be installed on node.
 
-Binaries should be placed in the directory listed in `$PATH` for `root` user on System Under Test (SUT) and Service machines.
- 
-#### Docker Image (only when experiments are going to be run on Kubernetes)
-User should create the Docker Image with all applications listed below available in `$PATH`. Image should be named `centos-swan-image` and should be based on Centos7 image. It should be available in local Docker images on SUT machine.
+**Docker**
+
+Please install Docker in version 17.03.
```

Comment:
Right. That's user guide, not for image developer, so I'll drop the magic and leave simple instructions from Docker support.